### DISCUSSION TITLE
how to install jans from source code?

### DISCUSSION THREAD

**Discussion by manojs1978** [JanssenProject/jans#1543] [Q&A] [answered]
how to install jans from source code?

i searched for instruction how to install jans from source code but didnot find any steps for this

**Comment by alexpereiramaranhao** [comment, marked as answer]
Hi @manojs1978 
There are a [PR 1054](https://github.com/JanssenProject/jans/pull/1054) open by @ossdhaval that I used to install janssen from source code

**Reply by manojs1978** [reply]
ok thanks
